src/prisma/schema.prisma

```diff
@@ -2086,3 +2087,15 @@ model CharmTransaction {
   toWallet   CharmWallet? @relation("toWallet", fields: [to], references: [id], onDelete: SetNull)
   metadata   Json?        @default("{}")
 }
+
+model FavoriteCredential {
+  id                 String            @id @default(uuid()) @db.Uuid
+  index              Int               @default(-1)
+  attestationId      String?
```

Comment:
Suggestion; let's add some comments to differentiate

// On chain
attestationId
chainId

// CharmVerse credential
issuedCredentialId
issuedCredential

// Gitcoin
gitcoinWalletAddress